app/panel/utils/utils.js

```diff
@@ -218,10 +218,11 @@ export function setTheme(doc, name, account) {
 	// First remove all other style elements which may be there
```

Comment:
Remove or correct this comment (and also let's move the previous one at the start of the function down to the appropriate place)

Reply:
Updated comment to "Get style tag for the active theme", and moved the first comment